src/wp-admin/js/custom-background.js

```diff
@@ -13,22 +26,46 @@
 			}
 		});
 
+		/**
+		 * @summary Alters the background size CSS property whenever the background size input has changed.
+		 *
+		 * @since 4.7
+		 */
 		$( 'select[name="background-size"]' ).change( function() {
 			bgImage.css( 'background-size', $( this ).val() );
 		});
 
+		/**
+		 * @summary Alters the background position CSS property whenever the background position input has changed.
+		 *
+		 * @since 4.7
+		 */
 		$( 'input[name="background-position"]' ).change( function() {
 			bgImage.css( 'background-position', $( this ).val() );
 		});
 
+		/**
+		 * @summary Alters the background repeat CSS property whenever the background repeat input has changed.
+		 *
+		 * @since 3.0
+		 */
 		$( 'input[name="background-repeat"]' ).change( function() {
 			bgImage.css( 'background-repeat', $( this ).is( ':checked' ) ? 'repeat' : 'no-repeat' );
 		});
 
+		/**
+		 * @summary Alters the background attachment CSS property whenever the background attachment input has changed.
+		 *
+		 * @since 4.7
+		 */
 		$( 'input[name="background-attachment"]' ).change( function() {
 			bgImage.css( 'background-attachment', $( this ).is( ':checked' ) ? 'scroll' : 'fixed' );
 		});
 
+		/**
+		 * @summery Binds the event for opening the WP Media dialog.
```

Comment:
Please fix typo: summery -> summary.
Please add a new line between @summary and @since